File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.py
T = int(input())

for i in range(T):
    numbers = list(map(int, input().split()))
    sum = 0
    minNumber = 100
    for number in numbers:
        if number % 2 == 0:
            sum += number
            if number < minNumber:
                minNumber = number

    print(sum)
    print(minNumber)

File: 백준/Bronze/3058. 짝수를 찾아라/짝수를 찾아라.js
const input = require('fs').readFileSync(0, 'utf-8').trim().split('\n');
const T = parseInt(input[0]);

for (let i = 1; i <= T; i++) {
    const numbers = input[i].split(' ').map(Number);
    let sumEven = 0;
    let minEven = Infinity;
    
    for (const num of numbers) {
        if (num % 2 === 0) {
            sumEven += num;
            if (num < minEven) {
                minEven = num;
            }
        }
    }
    
    console.log(sumEven);
    console.log(minEven);
}